### PR TITLE
CloudWatch: Add docs for Logs

### PR DESCRIPTION
**Pull request by aocenas**
Adds documentation section for CloudWatch logs

### CODE REVIEW

File: docs/sources/features/datasources/cloudwatch.md [hunk 1]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
...
+You can also write queries returning time series data by using [`stats` command](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_Insights-Visualizing-Log-Data.html). When using `stats` queries in Explore, you have to make sure you are in Logs Explore mode.
+
+TODO: image Explore switcher
+
+To the right of the query input field is CloudWatch Insights link that will open CloudWatch Insights console with your query so you can continue exploration there if needed.
+
+TODO: image highlight link
+
+### Using template variable
+TODO:
```
Comment (aocenas) [inline docs/sources/features/datasources/cloudwatch.md:186]: @kaydelaney Can you check this one? I assume this works but not exactly sure how. can you just reference the variables with the ${__var} syntax in the editor?

**Comment by aocenas** [comment]
@kaydelaney Can you also check the https://grafana.com/docs/grafana/latest/features/datasources/cloudwatch/#templated-queries section? this seems to be about template variables that are evaluated on cloudWatch side I assume. Is there something like that for Logs? Do we need to just mention that this works only for CloudWatch metrics?

File: docs/sources/features/datasources/cloudwatch.md [hunk 2]
```diff
@@ -110,6 +111,11 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
+## Using the Query Editor
+CloudWatch datasource can query data from both CloudWatch metrics and CloudWatch logs APIs and each has it's own specialised query editor. You select which API you want to query with QueryMode switch on top of the editor.
```
Comment (kaydelaney) [inline docs/sources/features/datasources/cloudwatch.md:115]:
```suggestion
CloudWatch datasource can query data from both CloudWatch metrics and CloudWatch logs APIs and each has its own specialized query editor. You select which API you want to query with using the query mode switch on top of the editor.
```

Just some small edits

File: docs/sources/features/datasources/cloudwatch.md [hunk 3]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
 
 This feature is not available for metrics that are based on metric math expressions.
 
+## Using the Logs Query Editor
+
+> Only available in Grafana v7.0+.
+
+To query CloudWatch logs you have select region and up to 20 log streams which you want to query. Use the main input area to write your query in [CloudWatch Logs Query Language](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
```
Comment (kaydelaney) [inline docs/sources/features/datasources/cloudwatch.md:175]:
```suggestion
To query CloudWatch logs you have to select the region and up to 20 log groups which you want to query. Use the main input area to write your query in [CloudWatch Logs Query Language](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
```

File: docs/sources/features/datasources/cloudwatch.md [hunk 4]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
...
+
+You can also write queries returning time series data by using [`stats` command](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_Insights-Visualizing-Log-Data.html). When using `stats` queries in Explore, you have to make sure you are in Logs Explore mode.
```
Comment (kaydelaney) [inline docs/sources/features/datasources/cloudwatch.md:177]:
```suggestion
You can also write queries returning time series data by using the [`stats` command](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_Insights-Visualizing-Log-Data.html). When using `stats` queries in Explore, you have to make sure you are in Metrics Explore mode.
```

File: docs/sources/features/datasources/cloudwatch.md [hunk 5]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
...
+
+TODO: image Explore switcher
+
+To the right of the query input field is CloudWatch Insights link that will open CloudWatch Insights console with your query so you can continue exploration there if needed.
```
Comment (kaydelaney) [inline docs/sources/features/datasources/cloudwatch.md:181]:
```suggestion
To the right of the query input field is a CloudWatch Insights link that will open the CloudWatch Insights console with your query so you can continue exploration there if needed.
```

File: docs/sources/features/datasources/cloudwatch.md (hunk 1 above)
Comment (kaydelaney) [reply docs/sources/features/datasources/cloudwatch.md:186]: Sure. I'll double check but yes I think you can use ${__var} syntax from within the editor.

File: docs/sources/features/datasources/cloudwatch.md [hunk 6]
```diff
@@ -304,6 +337,8 @@ Please note that for Grafana version 6.5 or higher, all API requests to GetMetri
 
 ## Service Quotas
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:338]: All headings should be in sentence case, not title case.

File: docs/sources/features/datasources/cloudwatch.md [hunk 7]
```diff
@@ -13,7 +13,8 @@ weight = 5
 
 # Using AWS CloudWatch in Grafana
 
-Grafana ships with built in support for CloudWatch. You just have to add it as a data source and you will be ready to build dashboards for your CloudWatch metrics.
+Grafana ships with built in support for CloudWatch. You just have to add it as a data source and you will be ready to
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:16]:
```suggestion
Grafana ships with built-in support for CloudWatch. Add it as a data source, then you are ready to
```

File: docs/sources/features/datasources/cloudwatch.md [hunk 8]
```diff
@@ -13,7 +13,8 @@ weight = 5
...
+ build dashboards or use explore with CloudWatch metrics and CloudWatch logs.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:17]:
```suggestion
 build dashboards or use Explore with CloudWatch metrics and CloudWatch logs.
```

File: docs/sources/features/datasources/cloudwatch.md (hunk 7 above)
Comment (oddlittlebird) [reply docs/sources/features/datasources/cloudwatch.md:16]: Please link to the Add a data source topic.

File: docs/sources/features/datasources/cloudwatch.md [hunk 9]
```diff
@@ -110,6 +111,11 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
+## Using the Query Editor
+CloudWatch datasource can query data from both CloudWatch metrics and CloudWatch logs APIs and each has its own specialized query editor. You select which API you want to query with using the query mode switch on top of the editor.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:115]: Per the Documentation style guide, "data source" is two words. Please apply throughout this topic.

File: docs/sources/features/datasources/cloudwatch.md [hunk 10]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
 
 This feature is not available for metrics that are based on metric math expressions.
 
+## Using the Logs Query Editor
+
+> Only available in Grafana v7.0+.
+
+To query CloudWatch logs you have to select the region and up to 20 log groups which you want to query. Use the main input area to write your query in [CloudWatch Logs Query Language](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:175]:
```suggestion
To query CloudWatch logs, select the region and up to 20 log groups which you want to query. Use the main input area to write your query in [CloudWatch Logs Query Language](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
```

File: docs/sources/features/datasources/cloudwatch.md [hunk 11]
```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
...
+
+You can also write queries returning time series data by using the [`stats` command](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_Insights-Visualizing-Log-Data.html). When making `stats` queries in Explore, you have to make sure you are in Metrics Explore mode.
+
+TODO: image Explore switcher
+
+To the right of the query input field is a CloudWatch Insights link that will open the CloudWatch Insights console with your query so you can continue exploration there if needed.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:181]:
```suggestion
To the right of the query input field is a CloudWatch Insights link that opens the CloudWatch Insights console with your query. You can continue exploration there if necessary.
```

File: docs/sources/features/datasources/cloudwatch.md [hunk 12]
```diff
@@ -304,6 +337,8 @@ Please note that for Grafana version 6.5 or higher, all API requests to GetMetri
 
 ## Service Quotas
 
+TODO: New service quotas with logs api?
+
 AWS defines quotas, or limits, for resources, actions, and items in your AWS account. Depending on the number of queries in your dashboard and the amount of users accessing the dashboard, you may reach the limit for the allowed number of CloudWatch GetMetricData requests per second. Note that quotas are defined per account and per region. If you're using multiple regions or have set up more than one CloudWatch data source to query against multiple accounts, you need to request a quota increase for each account and each region in which you hit the limit.
 
 To request a quota increase, visit the [AWS Service Quotas console](https://console.aws.amazon.com/servicequotas/home?r#!/services/monitoring/quotas/L-5E141212).
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:344]: @sfingerhut, do we want this linked in the documentation?

**Review by chrisdr — COMMENTED** [review]
Hello, I am the AWS technical writer for CloudWatch.  Please make a couple of small global changes, related to the fact that "CloudWatch Logs" is the official name of that service:

Change "CloudWatch logs"  to "CloudWatch Logs"   ("CloudWatch metrics" with a lowercase m is fine as-is)

Change "CloudWatch Insights"  to "CloudWatch Logs Insights"

Thanks!

**Review by kaydelaney — APPROVED** [review]

**Pull request merged**